src/hooks/useLocationContext.ts

```diff
@@ -17,6 +17,8 @@ export const useLocationContext = () => {
       const ns = `[a-z0-9-]+`;
       const resourceName = '[a-z0-9-.]+';
 
+      const params = new URLSearchParams(decodeURIComponent(location.search));
```

Comment:
could it be the case we have double decoding here? should it be

```suggestion
      const params = new URLSearchParams(location.search);
```